File: src/main/tests/input/simpleDeclaration.cpp
#include <iostream>
#include <string>
#include <vector>
#include <typeinfo>


int square(int x) { return x * x; }

int main() {

//    // Function call
    std::cout << "Square: " << square(5) << std::endl;

    // Explicit type construction
    std::string t{"world"};
    std::cout  << t << std::endl;

    // Post-increment and post-decrement
    int x = 10;
    std::cout << "x++: " << x++ << ", x--: " << x-- << std::endl;

    const int ci = 7;
    return 0;
}

File: src/main/tests/pythonGenerated/simpleDeclaration.py
import typing

def square(x) -> int:
    return x * x

def main() -> int:
    print("Square: ", square(5), sep="")
    print()
    t: str = "world"
    print(t, sep="")
    print()
    x: int = 10
    _t_0 = x
    x += 1
    _t_1 = x
    x -= 1
    print("x++: ", _t_0, ", x--: ", _t_1, sep="")
    print()
    ci = 7
    return 0

if __name__ == "__main__":
    main()
